procedures: starting a new workspace (#2227)
codenvy/che-docs: modules/user-guide/pages/url-parameter-for-the-devfile-file-path.adoc

:_content-type: CONCEPT
:description: URL parameter for the devfile file path
:keywords: devfile, file-path, devfile-path, devfile-file-path, url-parameter, devfile-path-parameter, devfile-file-path-parameter
:navtitle: URL parameter for the devfile file path
// :page-aliases:

[id="url-parameter-for-the-devfile-file-path_{context}"]
= URL parameter for the devfile file path

> When you visit a URL for starting a new workspace, {prod-short} searches the root directory of the linked Git repository for a devfile (with the file name `.devfile.yaml` or `devfile.yaml`). So the file path of the devfile in the linked Git repository must follow this convention.

Reviewer: ```suggestion
When you visit a URL for starting a new workspace, {prod-short} searches the root directory of the linked Git repository for a devfile (with the file name `.devfile.yaml` or `devfile.yaml`). Therefore, the file path of the devfile in the linked Git repository must follow the convention.
```
